src/main/java/com/extendedclip/deluxemenus/DeluxeMenus.java

```diff
@@ -5,14 +5,7 @@
 import com.extendedclip.deluxemenus.config.DeluxeMenusConfig;
 import com.extendedclip.deluxemenus.dupe.DupeFixer;
 import com.extendedclip.deluxemenus.dupe.MenuItemMarker;
-import com.extendedclip.deluxemenus.hooks.BaseHeadHook;
-import com.extendedclip.deluxemenus.hooks.HeadDatabaseHook;
-import com.extendedclip.deluxemenus.hooks.ItemHook;
-import com.extendedclip.deluxemenus.hooks.ItemsAdderHook;
-import com.extendedclip.deluxemenus.hooks.NamedHeadHook;
-import com.extendedclip.deluxemenus.hooks.OraxenHook;
-import com.extendedclip.deluxemenus.hooks.TextureHeadHook;
-import com.extendedclip.deluxemenus.hooks.VaultHook;
+import com.extendedclip.deluxemenus.hooks.*;

```

Comment:
The @Starmism import 😔

Reply:
IJ has a mind of its own. In its all-knowing greatness, it decided that there were too many individual imports!